news/2018-04-08-project-loved-week-of-april-8th.md

```diff
@@ -0,0 +1,279 @@
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/osu/kodama.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Camellia - kodama](https://osu.ppy.sh/beatmapsets/488016) by [ProfessionalBox](https://osu.ppy.sh/users/3250792) et al.  
+*written by [Kaifin](https://osu.ppy.sh/users/2596942)*
+
+[ProfessionalBox](https://osu.ppy.sh/users/3250792) has long been known as a high quality technical mapper, and [kodama](https://osu.ppy.sh/beatmapsets/488016) makes it very easy to see why. The map features gorgeously aesthetic slider shapes combined with velocity changes that perfectly represent the song and feel effortless to play, as well as some tricky spaced streams that are sure to test your aim control. I'd be remiss not to mention the guest difficulties of the set which are all top notch as well, particularly [-kevincela-](https://osu.ppy.sh/users/266596)'s Extra which is also a great take on the song!
+
+## <a name="osutaiko" id="osutaiko"></a>osu!taiko
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/%VIDEO_ID%?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+---
+
+*osu!taiko map descriptions written by [Backfire](https://osu.ppy.sh/users/263110)*
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/taiko/soundcloud-do-not-die.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Katagiri - Soundcloud, do not die!](https://osu.ppy.sh/beatmapsets/702806) by [Rafael Nadal](https://osu.ppy.sh/users/3456229) and [Sansheng](https://osu.ppy.sh/users/3579473)
+
+Boy, where do we have to start? Soundcloud, do not die! is a difficult beatmap by Rafael Nadal - not the famous tennisman - and is going to be very taxing on your stamina. It also comes with a slightly easier difficulty by Sansheng, complimenting the map and offering a different approach to the beatmap! Both of the difficulties are very difficult to master, as its density is very high, reaching 1,000 notes per minute. Needless to say, this beatmap is really not for the faint-hearted! If you're looking for a fun, difficult beatmap and want to train your stamina, look no further!
+
+[![](/wiki/shared/news/2018-04-08-project-loved-week-of-april-8th/taiko/grief-and-malice.jpg)](https://osu.ppy.sh/community/forums/topics/%TOPIC_ID%)
+
+[Helblinde - Grief & Malice](https://osu.ppy.sh/beatmapsets/439241) by [S a n d](https://osu.ppy.sh/users/676578)
+
+Created by S a n d and Backfire, Grief & Malice is a popular beatmap that some of you have already played in Taiko World Cup 2017. This awesome map uses mild SV, kiai fountains and many streams all set to a catchy tune! The map is challenging without being too hard, and is fun for most players who are seeking a great map to play anytime, in any occasion: multiplayer, singleplayer, tournaments, etc.
```

Comment:
If it was collabed with Backfire, shouldn't his name also appear on the banner and as one of the authors above this line?